docs/installation.rst

```diff
@@ -4,9 +4,17 @@ Installation
 
 At the command line::
 
+    $ pip install dateparser
+
+Or, if you don't have pip installed::
+
     $ easy_install dateparser
 
-Or, if you have virtualenvwrapper installed::
+If you want to install from the latest sources, you can do::
 
-    $ mkvirtualenv dateparser
-    $ pip install dateparser
+    $ git clone https://github.com/scrapinghub/dateparser.git
+    $ cd dateparser
+    $ python setup.py install
+
+Depending on how your environment is set up, you might need to use sudo for the
```

Comment:
Hmm, not sure. Users that are smart enough to figure out on their own probably don't need these docs, I was thinking it's nice to assume this is the first module they're installing. ;)